packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -83,6 +83,16 @@ const GMPArgsShape: TypedPattern<GmpArgsContractCall> = M.splitRecord({
   contractInvocationData: M.arrayOf(ContractCallShape),
 });
 
+const getCaipId = async (
+  orch: Orchestrator,
+  destinationEVMChain: AxelarChain,
```

Comment:
In this context, whether it's a destination or not seems irrelevant:

```suggestion
  chainName: AxelarChain,
```

This is minor. I'm mostly bothered by `destinationEVMChain`; it's such a mouthful. I like 2-word names.